test-suite/squidconf/tls-certs

```diff
@@ -0,0 +1,256 @@
+## Copyright (C) 1996-2018 The Squid Software Foundation and contributors
+##
+## Squid software is distributed under GPLv2+ license and includes
+## contributions from numerous individuals and organizations.
+## Please see the COPYING and CONTRIBUTORS files for details.
+##
+
+#
+# The following tests check the sparse matrix of (valid) configs:
+#
+#  A) tls-cert PEM contents:
+#    1) cert only           tls-cert=./leaf-key-rsa.pem
+#    2) full chain          tls-cert=./leaf-chain-rsa.pem
+#    3) root CA             tls-cert=./ca-root-rsa.pem
+#    4) incomplete chain    tls-cert=./ca-mid-rsa.pem
+#    5) missing
+#
+#  B) tls-key parameter:
+#    1) separate:       tls-cert=./leaf-rsa.crt tls-key=./leaf-rsa.pkey
+#    2) merged:         tls-cert=./leaf-key-rsa.pem
```

Comment:
Let's clarify that `tls-cert` contents comes from (A) rather than being fixed by this selector:

```suggestion
#    1) separate:       tls-cert=....crt tls-key=....pkey
#    2) merged:         tls-cert=....pem
```